charabia/src/normalizer/ae_oe_normalizer.rs.rs

```diff
@@ -0,0 +1,162 @@
+// Import `CharNormalizer` trait.
+use super::{ CharNormalizer, CharOrStr };
+use crate::Token;
+
+// Make a small documentation of the specialized Normalizer like below.
+/// <Script/Language> specialized [`Normalizer`].
+///
+/// This Normalizer uses [`<UsedLibraryToNormalize>`] internally to normalize the provided token.
+/// <OptionalAdditionnalExplanations>
+pub struct OE_AE_Normalizer;
+
+// All normalizers only need to implement the method `normalize_char` and the method `should_normalize` of the `CharNormalizer` trait.
+impl CharNormalizer for AE_OE_Normalizer {
+    // Creates the normalized version of the provided char.
+    // In this example we will remove whitespaces and lowercase other characters.
+    fn normalize_char(&self, c: char) -> Option<CharOrStr> {
+        match c {
+            'œ' => Some("oe".to_string().into()),
+            'æ' => Some("ae".to_string().into()),
+            'Œ' => Some("OE".to_string().into()),
+            'Æ' => Some("AE".to_string().into()),
+            _ => Some(c.into()),
+        }
+    }
+
+    // Returns `true` if the Normalizer should be used.
+    fn should_normalize(&self, token: &Token) -> bool {
+        // here we lowercase only on Latin and Cyrillic Scripts and if the current token contains an uppercased character.
+        token.script == Script::Latin &&
+            token.script == Script::Cyrillic &&
+            (token.lemma.chars().any('œ') ||
+                token.lemma.chars().any('æ') ||
+                token.lemma.chars().any('Œ') ||
+                token.lemma.chars().any('Æ'))
+    }
+}
+
+// Include the newly implemented Normalizer in the tokenization pipeline:
+//     - change the name of the file `dummy_example.rs` to `dummy.rs`
+//     - import module by adding `mod dummy;` (filename) in `normalizer/mod.rs`
+//     - Add Normalizer in `NORMALIZERS` in `normalizer/mod.rs`
+//     - check if it didn't break any test or benhchmark
```

Comment:
The normalizer is implemented, however, you didn't import it in the mod.rs file and added in `NORMALIZERS`, you should follow the bellow instructions and remove them from the file:
```suggestion
```

Reply:
Sure, just want to confirm one thing, should it be added in `NORMALIZERS` or `LOSSY_NORMALIZERS`?

Reply:
Hello @Soham1803,

deeply sorry for the delay,
`LOSSY_NORMALIZER` is better